blog/_posts/2023-04-18-integrating-scalajs-with-vite.md

```diff
@@ -0,0 +1,127 @@
+---
+layout: blog-detail
+post-type: blog
+by: Sébastien Doeraene
+title: Integrating Scala.js with Vite
+description: Using the new vite-plugin-scalajs to better integrate Scala.js with the Vite build tool.
+---
+
+One of the cornerstones of Scala.js is its ability to interoperate with JavaScript, in order to leverage its massive ecosystem.
+At the language level, we have good foundations and good tooling (namely [ScalablyTyped](https://scalablytyped.org/)) to reach out to JavaScript libraries.
+However, at the tooling level, Scala.js remains centered around Scala tooling.
+While this is good for the *coding* part of development, it is not so great for iterative feedback loops with the browser, or integration of other front-end assets.
```

Comment:
why “other” front-end assets, were any front-end assets mentioned yet? are assets, like, image files and such? or do you mean useful tooling?

omitting "other” is one possibility, but consider rewriting, because I actually have no idea what “integration of other front-end assets” even is. maybe be more explicit about what the actual problem is, even if it means you need more words in order to explain.